src/plugins/GUITestBase/src/tests/regression_scenarios/GTTestsRegressionScenarios_7001_8000.cpp

```diff
@@ -470,6 +471,69 @@ GUI_TEST_CLASS_DEFINITION(test_7152) {
     CHECK_SET_ERR(bottomRight == "11/40/35", "Bottom right position is wrong: " + bottomRight);
 }
 
+GUI_TEST_CLASS_DEFINITION(test_7161) {
+    class ItemPopupChooserByPosition : public PopupChooser {
+    //for some reason PopupChooser don not work properly, so we choose item by position
+    public:
+        ItemPopupChooserByPosition(HI::GUITestOpStatus &os, int _pos)
+            : PopupChooser(os, QStringList()), pos(_pos) {
+        }
+
+        virtual void run() override {
+            GTMouseDriver::release();
```

Comment:
Remove this release. Independent press/release events must be used with extra care because they may have cross test issues